integration-tests/scripts/setup-test-environment.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
```

Comment:
Could we do this in Ginkgo like out other stuff? What are checking here exactly, that out integration test pipeline runs?

Reply:
WRT PR description about this might be going elsewhere, I wouldn't want to extensively use bash there either if I can avoid it.